.gitignore

```diff
@@ -1,160 +1,45 @@
-# Byte-compiled / optimized / DLL files
+# This file is centrally managed as a template file in https://github.com/canonical/solutions-engineering-automation
+# To update the file:
+# - Edit it in the canonical/solutions-engineering-automation repository.
+# - Open a PR with the changes.
+# - When the PR merges, the soleng-terraform bot will open a PR to the target repositories with the changes.
+
+# Python Byte-compiled / optimized / DLL files
 __pycache__/
 *.py[cod]
 *$py.class
 
-# C extensions
-*.so
+# Test files and directories
+.pytest_cache/
+.coverage
+.tox
+reports/
+**/report/
+htmlcov/
+.mypy_cache
 
-# Distribution / packaging
-.Python
+# python virtual environments (for local dev)
+.venv
+venv
+env
+
+# Build artefacts
+output/
+.build/
 build/
-develop-eggs/
+*.charm
+*.snap
+# python build artefacts
+deb_dist/
 dist/
-downloads/
-eggs/
-.eggs/
-lib/
-lib64/
-parts/
-sdist/
-var/
-wheels/
-share/python-wheels/
 *.egg-info/
-.installed.cfg
-*.egg
-MANIFEST
-
-# PyInstaller
-#  Usually these files are written by a python script from a template
-#  before PyInstaller builds the exe, so as to inject date/other infos into it.
-*.manifest
-*.spec
 
-# Installer logs
-pip-log.txt
-pip-delete-this-directory.txt
-
-# Unit test / coverage reports
```

Comment:
A big different between these two files so I have some concern, please double check if it's correct or not. @samuelallan72

Reply:
Hmm yeah I didn't check this one - we can remove the file from automation for this repo if it's problematic.  However, a quick glance (I'll look more tomorrow) seems like this is from a large template, so there are many files here that didn't need to be anyway.

Reply:
I think this is fine :crossed_fingers: